example/settings.py

```diff
@@ -97,3 +99,26 @@
     }
 
 STATICFILES_DIRS = [os.path.join(BASE_DIR, "example", "static")]
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+        },
+    },
+    "root": {
+        "handlers": ["console"],
+        "level": "WARNING",
+    },
+    "loggers": {
+        # Log when an asynchronous handler is adapted for middleware.
+        # See warning here: https://docs.djangoproject.com/en/4.2/topics/async/#async-views
```

Comment:
Sorry, I don't fully understand the purpose here. Can you help me understand this?

Reply:
Warning the user when having to switch from async to sync in the middleware/view stack makes a lot of sense.

I'm not 100% sure if that has to be a part of django-debug-toolbar's example settings though. Is it necessary to define `LOGGING` at all? Isn't it the default behavior already to emit those warnings?

Reply:
This allows surfacing the fact that django-debug-toolbar is not an async middleware, therefore Django needs to some extra stuff when using async views. 

Probably only relevant until django-debug-toolbar becomes an async middleware.

Shall we remove from the example then?

Reply:
I think my vote is remove it or keep it but default it to DEBUG so that we get that logging you mention.

Reply:
I am going to remove this from the PR since #1938 makes the middleware async.